src/main/resources/com/puppycrawl/tools/checkstyle/grammars/javadoc/JavadocParser.g4

```diff
@@ -861,39 +861,41 @@ description: (
 
 reference:
       (
-            PACKAGE (DOT | CLASS)* HASH? MEMBER? parameters?
-            | (DOT | CLASS)+ HASH? MEMBER? parameters?
+            PACKAGE_CLASS HASH? MEMBER? parameters?
```

Comment:
there are already examples like: `@see lowercase.UPPERCASE.CamelCase.packageName.ClassName#fn` which are equivalent to the above one, aren't they ?